qualtran/bloqs/msft_resource_estimator_interop.ipynb

```diff
@@ -0,0 +1,136 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from qualtran.bloqs.factoring.mod_exp import ModExp\n",
+    "from qualtran.drawing import show_bloq\n",
+    "\n",
+    "N = 13*17 # integer to factor\n",
+    "g = 8\n",
+    "\n",
+    "mod_exp = ModExp(base=g, mod=N, exp_bitsize=32, x_bitsize=32)\n",
+    "show_bloq(mod_exp)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
```

Comment:
Can you add a markdown cell with some more details about the interface (i.e. Microsoft's resource estimator expects a `LogicalCounts` dictionary, which can be populated using gate and qubit counts obtained via a Bloq). Also, it'd be great to add some links to documentation and references from Microsoft's azure resource estimator, that people can look at to figure out which other fields can be populated for LogicalCounts dictionary

Reply:
Done